OpenSim/Simulation/Manager/Manager.h

```diff
@@ -173,11 +167,25 @@ class OSIMSIMULATION_API Manager
     { _writeToStorage =  writeToStorage; }
 
     // Integrator
+    enum class Integrator {
+        CPodes,
+        ExplicitEuler,
+        RungeKutta2,
+        RungeKutta3,
+        RungeKuttaFeldberg,
+        RungeKuttaMerson,
+        //SemiExplicitEuler, no error ctrl, requires fixed stepSize arg on construction
+        SemiExplcitEuler2,
+        Verlet
+    };
```

Comment:
It would be nice to add doxygen documentation that links to each of these integrators in Simbody. I think the doxygen output will look nice if you do the following:

```
    ...
    /** For details, see SimTK::CPodesIntegrator. */
    CPodes,
    ...
```